anesthetic/utils.py

```diff
@@ -76,7 +76,7 @@ def quantile(a, q, w=None, interpolation='linear'):
     w = np.array(list(w))  # Necessary to convert pandas arrays
     i = np.argsort(a)
     c = np.cumsum(w[i[1:]]+w[i[:-1]])
-    c /= c[-1]
+    c = c/c[-1]
```

Comment:
Needed to make this trivial change to avoid an error if the weights are integers and the `numpy.ufunc` complaints.

Reply:
Yes, I've got this one in a few branches, too.